api/views/course_admin.py

```diff
@@ -40,6 +49,37 @@ def register_user(self, request, pk=None):
         if course.is_registered(user):
             raise ValidationError(ERROR_MESSAGES.COURSE_REGISTRATION.ALREADY_REGISTERED)
 
-        # TODO: Implement the registration
+        # TODO: Implement the registration that uses of canvas_user_id which can be done from canvas api
+        canvascourseregistration = CanvasCourseRegistration(course=course, user=user)
```

Comment:
Registering a user without setting the canvas_user_id will break the functionality. Please talk to bowen and figure out a way to do it. My idea is to pass the student id for this and find the canvas user based on the student id.

Reply:
Done. Remove the register_user function according to the discussion with Bowen.